File: beecrowd/1/1019/main.go
package main

import (
	"fmt"
)

func main() {
	var segundos int
	var tempo = [3]int{60 * 60, 60, 1}
	fmt.Scan(&segundos)
	for i, v := range tempo {
		tempo[i] = segundos / v
		segundos -= tempo[i] * v
	}
	fmt.Printf("%d:%d:%d\n", tempo[0], tempo[1], tempo[2])
}

File: beecrowd/1/1019/main.cpp
#include <iostream>

int main(void) {
    int segundos = 0;
    int tempo[3] = {60 * 60, 60, 1};
    std::cin >> segundos;
    for (int i = 0; i < 3; i++) {
        int v = tempo[i];
        tempo[i] = segundos / v;
        segundos -= tempo[i] * v;
    }
    std::cout << tempo[0] << ":" << tempo[1] << ":" << tempo[2] << std::endl;
    return 0;
}
